src/entities/ShareClass.ts

```diff
@@ -1014,6 +1014,13 @@ export class ShareClass extends Entity {
     }, this.pool.chainId)
   }
 
+  /**
+   * Retrieve members of the share class.
+   */
+  members() {
```

Comment:
`holders` might be a better name. with the deRWA assets being freely tradeable, holders don't have to be members